### PR TITLE
Feature: Adding equals/hashcode methods and test coverage to UserOrg model

### PR DESCRIPTION
**Pull request by brianmuse**
See title.

### CODE REVIEW

File: src/main/java/com/robinpowered/sdk/model/UserOrganization.java [hunk 1]
```diff
@@ -1,9 +1,10 @@
 package com.robinpowered.sdk.model;
 
+import com.google.common.base.Objects;
 import org.joda.time.DateTime;
 
 /**
- * An {@link Organization} containing {@link UserAccess} data.
+ * An {@link Organization} entity that represents a user's membership, containing the {@link UserAccess} model.
```
Comment (christophermark) [inline src/main/java/com/robinpowered/sdk/model/UserOrganization.java:7]: _represents_, or _contains_?

File: src/main/java/com/robinpowered/sdk/model/UserOrganization.java [hunk 2]
```diff
@@ -32,17 +33,31 @@ public UserOrganization(int id, String slug, boolean isOrganization, DateTime cr
...
+        if (!super.equals(o)) return false;
+        UserOrganization that = (UserOrganization) o;
+        return Objects.equal(userAccess, that.userAccess);
     }
 
-    public UserAccess getUserAccess() {
-        return userAccess;
+    @Override
+    public int hashCode() {
+        return Objects.hashCode(super.hashCode(), userAccess);
```
Comment (christophermark) [inline src/main/java/com/robinpowered/sdk/model/UserOrganization.java:56]: Is taking a hash of a hash going to cause problems? Or should we be doing:

```
return Objects.hashCode(super, userAccess);
```

File: src/main/java/com/robinpowered/sdk/model/UserOrganization.java (hunk 1 above)
Comment (brianmuse) [inline src/main/java/com/robinpowered/sdk/model/UserOrganization.java:7]: represents

File: src/main/java/com/robinpowered/sdk/model/UserOrganization.java (hunk 2 above)
Comment (brianmuse) [inline src/main/java/com/robinpowered/sdk/model/UserOrganization.java:56]: This is how it should be.

**Comment by christophermark** [comment]
:+1:

**Pull request merged**